k8s/wrappers/services/containerd

```diff
@@ -7,7 +7,18 @@ k8s::common::setup_env
 # Ensure XDG_RUNTIME_DIR exists, needed by runc for 'kubectl exec' to work
 [ ! -z "${XDG_RUNTIME_DIR:=}" ] && mkdir -p "${XDG_RUNTIME_DIR}"
 
-# Apply the containerd profile
-k8s::common::is_strict && apparmor_parser -r "$SNAP/k8s/profiles/containerd"
+# Apply apparmor profile for "cri-containerd.apparmor.d"
+if ! apparmor_parser -r "$SNAP/k8s/profiles/containerd"; then
+  echo "
```

Comment:
Hm, should we just not start the containerd service instead?
As a user I'd rather have a failing service when bringing up the cluster than subtle failures later. 
I'd also say that this message will likely be not seen by the user. At least I wouldn't checkt the journal log if all services come up just fine.
WDYT?

Reply:
so, we might be on a system where AppArmor is not available or enabled, and on such systems a failure here is expected, which is why I started with this "loud silent" warning on the logs.

Reply:
I see, that makes sense. Thanks for the explanation!